arrows/ffmpeg/ffmpeg_init.cxx

```diff
@@ -23,47 +23,50 @@ std::mutex ffmpeg_init_mutex;
 
 static auto ffmpeg_logger = kwiver::vital::get_logger( "arrows.ffmpeg" );
 
-//-----------------------------------------------------------------------------
+// ----------------------------------------------------------------------------
 void
-ffmpeg_kwiver_log_callback(void* ptr, int level, const char* fmt, va_list vl)
+ffmpeg_kwiver_log_callback( void* ptr, int level, const char* fmt, va_list vl )
 {
   static int print_prefix = 0;
-  char line[1024];
-  av_log_format_line(ptr, level, fmt, vl, line, sizeof(line), &print_prefix);
-  std::string msg(line);
-  kwiver::vital::right_trim(msg);
-  switch(level)
+  char line[ 1024 ];
+  av_log_format_line( ptr, level, fmt, vl, line, sizeof( line ),
+                      &print_prefix );
+
```

Comment:
JFYI, there's an option about adding a blank before variable declaration blocks. I have somewhat mixed feelings about that, but for now have been letting it do its thing. (This isn't a request for change, just letting folks know why Uncrustify added a blank here.)